vpr/src/route/route_timing.h

```diff
@@ -16,6 +16,8 @@
 
 extern bool f_router_debug;
 
+extern bool f_router_debug;
```

Comment:
This is a merge error, this is doubled up now

Reply:
Done